### PR TITLE
POSIX shell

### PR DESCRIPTION
**Pull request by aosync**
Added the POSIX shell version :sunglasses:

### CODE REVIEW

File: sh.sh [hunk 1]
```diff
@@ -0,0 +1,32 @@
+read _ _ KERNEL _ </proc/version
+
+[ -z "$HOSTNAME" ] && read HOSTNAME </etc/hostname
+
+TASKS=0
+for dir in /proc/*; do
+  case $dir in
+  	*[0-9]*) TASKS=$((TASKS + 1)) ;;
```
Comment (aeosynth) [inline sh.sh]: does this actually work, would it incorrectly match `x1` or `1x`? is it testing if a name contains only numbers, or contains any numbers?

File: sh.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
+read _ _ KERNEL _ </proc/version
+
+[ -z "$HOSTNAME" ] && read HOSTNAME </etc/hostname
```
Comment (aeosynth) [inline sh.sh]: don't need to test for hostname, testing is ignored everywhere. go/rust would be twice as long if we did test.

File: sh.sh [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+done
+
+set --
+
+while IFS=$'\n' read -r l; do
+	set -- $@ $l
+done </proc/meminfo
+
+TOTAL=$(($2 / 1000))
+AVAIL=$(($8 / 1000))
```
Comment (aeosynth) [inline sh.sh]: the 2 and 8 are too magic, sorry, try something else. i guess knowing to use the first and third lines is also magaical, easier to verify at a glance.

**Review by aeosynth — COMMENTED** [review]
feel free to review the bash code. as in, if you can improve it, pr welcome.

File: sh.sh (hunk 1 above)
Comment (aosync) [reply sh.sh]: You are right, idk where I had my mind. I have found another reliable way to do it by testing whether comparing the process id with -1 fails

**Pull request merged**